### PR TITLE
Improve code quality

### LINKED ISSUE #95: Add more passenger profiles to initialised json list

### PR DESCRIPTION
**Pull request by JoelHo**

### CODE REVIEW

**Review by markmcwong — COMMENTED** [review]
LGTM

File: src/test/java/seedu/address/logic/parser/PoolCommandParserTest.java [hunk 1]
```diff
@@ -128,11 +129,9 @@ public void parse_compulsoryFieldMissing_failure() {
...
-
-        //        // missing one commuter prefix
-        //        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + TRIPDAY_DESC_MONDAY
-        //                + TRIPTIME_DESC_MORNING + VALID_COMMUTER_1 + COMMUTER_DESC_2, expectedMessage);
+        // missing one commuter prefix
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + TRIPDAY_DESC_MONDAY
+                + TRIPTIME_DESC_MORNING + VALID_COMMUTER_1, expectedMessage);
 
         //        // missing one commuter prefix
         //        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + TRIPDAY_DESC_MONDAY
```
Comment (chewterence) [inline src/test/java/seedu/address/logic/parser/PoolCommandParserTest.java]: there still appears to be commented out code

**Review by chewterence — CHANGES_REQUESTED** [review]

**Comment by chewterence** [comment]
LGTM

**Comment by chewterence** [comment]
LGTM

**Comment by JoelHo** [comment]
Closes #95

**Pull request merged**